mlflow/R/mlflow/R/tracking-globals.R

```diff
@@ -24,6 +31,8 @@ mlflow_get_active_experiment_id <- function() {
 #' @export
 mlflow_set_tracking_uri <- function(uri) {
   .globals$tracking_uri <- uri
+  mlflow_register_tracking_event("tracking_uri", list(uri = uri))
```

Comment:
Should this be named e.g. "set_tracking_uri" for parity with events like "log_metric"?

Reply:
My personal preference is to prefix absolutely everything, we've done that in packages like `sparklyr` (as in `spark_`, `ml_`, etc), `tensorflow` (as in `tf_`, etc), and more recently `torch` (as in `torch_`, etc.). The reasoning, not having a prefix in R packages is expensive... Once you define something outside of a package, you need to figure out how that might potentially interact with other packages.

For instance, for `log_metric`, we might have to consider how that interacts with other model logging methods from the R community. So far, I don't think there are any package clashes, but one day this could change. Would be nice for instance if one could `log_metric` without using mlflow, then connect mlflow, and the code won't break.

So net-net, I would encourage we keep prefixes in most functions, unless we feel strongly a function is meant to be top-level and deserves the extra work of potentially interfacing with other packages.